### DISCUSSION TITLE
What does the 'score' column refer to in the dataframe returned by `get_city_station_options`?

### DISCUSSION THREAD

**Discussion by Milind220** [Ozon3Org/Ozon3#108] [Q&A] [answered]
<img width="929" alt="Screenshot 2022-04-13 at 12 20 10 PM" src="https://user-images.githubusercontent.com/68847270/163099419-49998419-1a32-4b53-b6c0-9ae29e4cd15b.png">

On the far right...

@lahdjirayhan You may be able to answer this :)

**Comment by lahdjirayhan** [comment, marked as answer]
It's maybe some sort of "relevance" or "match" score, respective to the entered search query. It is included in the backend-sent data so that the website's frontend can just show which entry to be the top/first/most likely match. Might as well just display it for completeness, I thought. :)

Hope this helps!
